api/node/routes.go

```diff
@@ -51,6 +51,7 @@ func Routes(router *gin.RouterGroup) {
 	router.GET("/heartbeatstatus", HeartbeatStatus)
 	router.GET("/statistics", Statistics)
 	router.GET("/status", StatusMetrics)
+	// testing section
```

Comment:
might change this comment as if you leave it like this, the next person will think this is a testing section, but it's the real one instead. Maybe put something like `// placeholder for custom routes`

Reply:
done